example-apps/html-js/index.html

```
@@ -0,0 +1,121 @@
+      <div class="navbar__items">
+        <a class="navbar__brand" href="https://img2num.dev">
+          <img src="https://img2num.dev/img/favicon.svg" alt="Logo" class="navbar__logo" />
+          <b class="navbar__title">Img2Num</b>
+        </a>
+        <a class="navbar__item navbar__link" href="https://img2num.dev/docs/">Docs</a>
+        <a class="navbar__item navbar__link" href="https://img2num.dev/changelog/">Changelog</a>
+      </div>
+      <div class="navbar__items navbar__items--right">
+        <a class="navbar__item navbar__link" href="https://github.com/Ryan-Millard/Img2Num">GitHub</a>
+        <a class="navbar__item navbar__link" href="https://www.npmjs.com/package/img2num">npm</a>
+        <button id="themeToggle" class="button button--outline button--sm navbar__item">🌙</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)